application/schemas/artifact_schema.py

```diff
@@ -19,10 +20,5 @@ class ArtifactSchema(BaseSchema):
     @output_decorator
     def transform_fields(self, data):
         """ Transforms field for output """
-        data["url"] = self.build_url(data.pop("file_url"))
+        data["url"] = build_url(data.pop("file_url"))
         return data
-
-    def build_url(self, file_url):
-        """ Schema: fileserver/id_filename """
-        return current_app.config["FILE_SERVER"] + \
-                "/" + file_url
```

Comment:
This I would just mark as a class method and then it can stay in here.

Reply:
This is now a static method in artifact schema.

Reply:
Again I'd prefer a class method here, althought this one is supposed to be public so it doesn't really matter. Just my preference.